tests/console/suseconnect_scc.pm

```diff
@@ -39,7 +39,8 @@ sub run {
 
     select_console('root-console');
     assert_script_run $cmd;
-    assert_script_run 'SUSEConnect --list-extensions';
+    assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mServer Applications Module\')"';
+    assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mWeb and Scripting Module\')"';
```

Comment:
You are not checking here, if the Web and Scripting module has a further indentation.
You need to use 4? more spaces here:

```suggestion
    assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'        \e\[1mWeb and Scripting Module\')"';
```